.deployignore

```diff
@@ -62,6 +62,7 @@ yarn.lock
 /cron-control-next/bin/
 /cron-control-next/runner/
 /cron-control-next/tests/
+/search/data/
```

Comment:
Not sure this is necessary since the new file is under `tests/` which is already `.deployignore`-d?

```suggestion
```